module/zfs/dbuf.c

```diff
@@ -1436,7 +1452,7 @@ dbuf_read_verify_dnode_crypt(dmu_buf_impl_t *db, uint32_t flags)
  */
```

Comment:
The convention seems to be that the tag always comes last.

Reply:
I went ahead and fixed this, so it follows the convention of everything else.